spock-core/core.gradle

```diff
@@ -49,20 +47,21 @@ jar {
       'Implementation-Vendor': 'spockframework.org',
       'Automatic-Module-Name': 'org.spockframework.core'
     )
-    instruction 'Export-Package', 'org.spockframework.*', 'spock.*'
-    instruction 'Embed-Dependency', 'groovy;inline=false', 'junit;inline=false'
-    instruction 'Import-Package',
-        'groovy.*;version="[2.4,4)"',
-        'org.codehaus.groovy.*;version="[2.4,4)"',
-        'org.junit.*;version="[4,6)"',
-        'org.hamcrest.*;version="[1,2)";resolution:=optional',
-        'org.objenesis.*;version="[2,3)";resolution:=optional',
-        'org.apache.tools.ant.*;version="[1.7,2)";resolution:=optional',
-        'net.bytebuddy.*;version="[1.9,2)";resolution:=optional',
-        'net.sf.cglib.*;version="[3.2,4)";resolution:=optional',
-        'org.objectweb.asm.*;version="[5.2,6)";resolution:=optional',
-        'org.w3c.dom.*'
   }
+  bnd(
+    'Export-Package': ['org.spockframework.*', 'spock.*'].join(','),
+    'Embed-Dependency': ['groovy;inline=false', 'junit;inline=false'].join(','),
+    'Import-Package': [
+      "groovy.*;version=\"[${groovyVersion},${org.gradle.util.VersionNumber.parse(groovyVersion).major + 1})\"",
```

Comment:
You might consider using either slashy strings or triple-quoted strings to remove the need to escape the double quotes. As there are double-quotes as last characters probably better slashy strings.